### PR TITLE
Minor model changes

### PR DESCRIPTION
**Pull request by sleshchenko**
1. Add CloudShell example workspace configuration.
2. Update examples CRs to reflect the actual model state.
3. Minor model changes:
  - fields order;
 - DevFileSpec -> DevfileSpec;
 - extracting possible values for strings to costs;
 - and a few more minor;
4. Regenerate openapi and k8s according to minor model changes.
5. Code clean up:
  - Change default hard-coded pull policy to `Always` since it's safer. I mean it needed some time to me to understand why the image that was supposed to be fixed is still failing );
  - Move some strings values to const file, like WorkspaceIDLabel, WorkspaceNameLabel, ...;
  - Fixed some of IDE's warning: removed redundant type declaration, added error checking, ...;
  - Fix typos and translates comments to English;
  - Move CheRest related things to a separate package since it's not really needed for CloudSell;
  - Hardcode UUID to 1234 to make CloudShell working. Later it makes sense to make this value configurable as in Che Server;
6. Configure codeowners and pull request template for GitHub.

It's done in the scope of https://github.com/eclipse/che/issues/15783

It's tested on minikube (cloud shell workspace with the nightly plugin-registry)

### CODE REVIEW

**Comment by amisevsk** [comment]
I'll open a follow-up PR with my suggestions so we can discuss.

File: pkg/controller/workspace/che_rest/che_rest_apis.go [hunk 1]
```diff
@@ -25,29 +25,29 @@ import (
...
 )
 
-func addCheRestApis(wkspProps WorkspaceProperties, podSpec *corev1.PodSpec) ([]runtime.Object, string, error) {
+func AddCheRestApis(wkspProps WorkspaceProperties, podSpec *corev1.PodSpec) ([]runtime.Object, string, error) {
 	cheRestApisPort := 9999
 	containerName := "che-rest-apis"
 	podSpec.Containers = append(podSpec.Containers, corev1.Container{
 		Image:           ControllerCfg.GetCheRestApisDockerImage(),
-		ImagePullPolicy: corev1.PullIfNotPresent,
+		ImagePullPolicy: corev1.PullAlways,
```
[the remark this answers is not in the document]
Comment (sleshchenko) [reply pkg/controller/workspace/che_rest/che_rest_apis.go]: Agree. Will add an ability to configure pull policy with Controller Config Map

**Comment by sleshchenko** [comment]
> Did you also test it on OpenShift 4 (OLM installation) ?

@davidfestal No, I never run devworkspace controller via OLM installation.
Do you have something concrete why it may stop working?

**Comment by davidfestal** [comment]
Not necessarily. But sine it is simple to test, I was wondering if something prevented this test.

**Pull request merged**